### ISSUE TITLE
Review Like Feature

### ISSUE THREAD

**Issue by ohager** [signum-nft/feedback#56]
**Is your feature request related to a problem? Please describe.**
There is some noise in the community regarding the "Like" feature. At the moment of writing, there's no rule for liking an NFTs, i.e. it can be easily abused. This actually violates the ToS:

>Engage in behaviors that have the intention or the effect of artificially causing an item or collection to appear at the top of search results, or artificially increasing view counts, favorites, or other metrics that NFTPortal might use to sort search results.

The most liked NFTs are exposed on the landing page, and appear on top when sorted. 

**Describe the solution you'd like**
Likes are controlled by a smart contract, but it doesn't keeps not track of the users who like. This is written into the blockchain, and can be extracted only from there. Blocking a single user makes not much sense, as account creation can be automated also.
The solution should be able to eliminate manipulation and/or any significant advantage and somehow must not discredit the owner or creator (in case s/he did not manipulated the likes) of an NFT. 

**Describe alternatives you've considered**
After some thinking, I tend to remove at least the most likes Nfts on the landing page to reduce the exposure.

**Comment by frankTheTank72** [comment]
Maybe we could shuffle the list of liked NFTs like for other lists

**Comment by paulpoco** [comment]
Not new, many of us pointed this out before.

The portal could see the likes, when it see a 2nd or 3rd or 4th etc like then if an address liked something multiple times just just don't include the extra likes on the NFT page likes.

https://github.com/signum-nft/feedback-signumart/issues/44

**Comment by frankTheTank72** [comment]
> Not new, many of us pointed this out before.
> 
> The portal could see the likes, when it see a 2nd or 3rd or 4th etc like then if an address liked something multiple times just just don't include the extra likes on the NFT page likes.
> 
> #44

Yes i think finally we could only address this problem outside the smart contract - as the portal is doing screening on the "like-tx" like you said and doing some magic with filters and stuff

**Comment by Nivok-Spilkommen** [comment]
Maybe the 'least harm' approach would be to have a mechanism to remove all likes from suspicious NFTs.

**Comment by frankTheTank72** [comment]
we could work with the new map feature - the contract will just cost more in the execution
